slurm/2_ApplicationSpecific/VASP-GPU-podman/build_vasp.bash

```diff
@@ -112,16 +118,13 @@ then
         base_dir="/projects/rpci/${grp_group}/$(id -un)/VASP"
```

Comment:
I would recommend removing anything related to rpci.  This setup is an example for academic users who've purchased a vasp license.  If someone at Roswell ends up trying to use it, they will need to modify other parts of it anyway.

Reply:
done